test/test.env

```diff
@@ -15,3 +15,9 @@ LOGIN_ATTEMPT_CAP=2
 DISABLE_REQUEST_LOGGING=true
 NODE_TEST_MODE=true
 NODE_ENV=test
+GOCARDLESS_TEST_CLIENT_ID=cjchuyewhdj23d8279dh29test
+GOCARDLESS_TEST_CLIENT_SECRET=nco2i3914yu1icbckjbeco3test
+GOCARDLESS_LIVE_CLIENT_ID=acsfhuyewbvd3d8279dh29live
+GOCARDLESS_LIVE_CLIENT_SECRET=saq2i3914yu1icbckjbeco3live
```

Comment:
Where do we use these values? I suppose they are not valid secrets?

Reply:
Nope, no valid secrets. Replaced them with explicit values stating they are not secret, sorry about the confusion.